README.md

```diff
@@ -213,6 +213,12 @@ Tags are fetched in order to correctly generate versioning metadata.
 To view code coverage results open `coverage/index.html` in your browser
 after running `npm run test`.
 
+> If you run `npm run test` and the tests are running but the coverage report is not getting generated
```

Comment:
Is there a reason to use quote here? Don't think we use it before.

Reply:
I used a quote here to bring attention to the unexpected behavior. Let me know if you want me to change it.

Reply:
Quote means you copy it from somewhere not to highlighting though.

Reply:
I have removed the quote.